lib/versioner/systemtrash_darwin.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2016 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//+build darwin,cgo
```

Comment:
I don't know, I don't really have an issue with some functionality not necessarily being available in all build modes. We're going to run headfirst into this when we get to integrating inotify for example, as that's going to be all cgo all the time. Binaries created by cross compilation without cgo will simply lack this functionality.